nodeprobe/nodeprobe.go

```diff
@@ -114,15 +114,12 @@ func (p *Prober) probe(ctx context.Context) {
 	p.ready.Store(allNodesReady.Load())
 
 	if !p.ready.Load() {
-		p.logger.Debug("not all nodes are ready")
+		p.logger.Warn("not all nodes are ready")
```

Comment:
Should this be ERROR? Logs above use error when not ready:

```go
			if err != nil {
				logger.Error("failed to check if node is ready", zap.Error(err))
			} else if !ready {
				logger.Error("node is not ready")
			} else {
				logger.Info("node is ready")
			}
```

Reply:
@nkryuchkov changed to error